server/utils/firebase/changes.ts

```diff
@@ -0,0 +1,31 @@
+import { Step } from 'prosemirror-transform';
+import { Reference } from '@firebase/database-types';
+import { uncompressStepJSON } from 'prosemirror-compress-pubpub';
+
+import { editorSchema } from '../firebaseAdmin';
+
+export const getStepsInChangeRange = async (
```

Comment:
We will use this in an upcoming PR but it felt thematically appropriate to include here